src/components/grid.scss

```diff
@@ -120,45 +188,40 @@ $grid-breakpoints: $breakpoints;
    Columns elements
 \*============================================================================*/
 
-.grid,
 [class*='grid__col-'] {
   min-height: 1px;
 
+  .grid__row--stretch > & {
+    display: flex;
```

Comment:
Should flexbox stuff be in a different component ?
I mean, flex utiliies are not only used for grid.

Reply:
There is a ticket to create a list of flex helper classes, have a look at #14  😉

Reply:
Yes i saw :)
In my opinion `flex` & `grid` should be two different components

- To avoid having a monster `grid` component.
- `flex` is often use for non `grid` related elements

What do you think ?

Reply:
I think it is interesting to keep these small flex modifiers in the grid so they can be more precise in their properties, but a `_flex.scss` element in the framework is indeed necessary!